src/Npgsql/NpgsqlException.cs

```diff
@@ -42,8 +42,12 @@ public NpgsqlException(string? message)
         /// Specifies whether the exception is considered transient, that is, whether retrying the operation could
         /// succeed (e.g. a network error or a timeout).
         /// </summary>
-        public virtual bool IsTransient =>
-            InnerException is IOException || InnerException is SocketException || InnerException is TimeoutException;
+#if NET461 || NETSTANDARD2_0 || NETSTANDARD2_1 || NETCOREAPP3_0
+        public virtual bool IsTransient
+#else
+        public override bool IsTransient
+#endif
+            => InnerException is IOException || InnerException is SocketException || InnerException is TimeoutException;
```

Comment:
PostgresException with SQLSTATE = 40001 or 40P01 are probably the archetypes of transient exceptions in a database.

Reply:
Yep, they're included in the [PostgresException override of this property](https://github.com/npgsql/npgsql/blob/3f525e4a359be05bfc17b4ec3590333efd05908f/src/Npgsql/PostgresException.cs#L195) - but 40P01 is missing (and I agree should be there).

Do you want to do a quick pass on the current list and update it as appropriate?

Reply:
Yes. I'll open an issue so that I don't forget it.